themes/bootstrap3/js/preview.js

```diff
@@ -43,7 +43,7 @@ function applyPreviewUrl($link, url) {
 
     // Update associated record thumbnail, if any:
   $link.parents('.result,.record')
-        .find('.recordcover').parents('a').attr('href', url);
+        .find('.recordcover[data-linkpreview="true"]').parents('a').attr('href', url);
```

Comment:
I guess we have no need to squabble over milliseconds. And if someone wanted to style it they could use the same selector. Fine as is.